src/typ/tests.rs

```diff
@@ -1773,6 +1773,7 @@ fn infer_module_warning_test() {
             ast.name = vec!["my_module".to_string()];
             let (_, warnings) = infer_module(ast, &HashMap::new());
 
+            assert_eq!(true, warnings.len() > 0);
```

Comment:
Can write `assert!(warning.len() > 0)`, I think it might have a slightly clearer error message.

Reply:
Yeah that would be loads better, will do.